README.md

```diff
@@ -38,6 +39,7 @@
 ## Tools
 - [Visitor Badge](https://visitor-badge.glitch.me/#docs) - Count visitors for your README.md, Issues, PRs in GitHub
 - [Simple Icons](https://github.com/simple-icons/simple-icons#cdn-usage) -  SVG icons for popular brands for your README.md files
+- [Shields Project](https://shields.io/) - Use Shields to create profile badges (Compatible with Simple Icons)
```

Comment:
@ABSphreak This is a nice tool, I think adding this in 2nd place would be better, and also remove the brackets `(Compatible with Simple Icons)`

Reply:
Even visitors badge can be created from here, this is a bigger project than that, anyway, I will add it to 2nd as you requested.